cmd/execution_builder.go

```diff
@@ -158,6 +160,9 @@ func (builder *ExecutionNodeBuilder) LoadComponentsAndModules() {
 		AdminCommand("trigger-checkpoint", func(config *NodeConfig) commands.AdminCommand {
 			return executionCommands.NewTriggerCheckpointCommand(exeNode.toTriggerCheckpoint)
 		}).
+		AdminCommand("stop-at-height", func(config *NodeConfig) commands.AdminCommand {
+			return executionCommands.NewStopAtHeightCommand(exeNode.stopAtHeight)
+		}).
```

Comment:
Do we need to add this to VN as well?

Reply:
VN doesn't have admin commands. You can specify stop height by command line, since it restarts fast.